lib/poteto/post_review.rb

```diff
@@ -1,29 +1,34 @@
 # frozen_string_literal: true
 
+require 'octokit'
+
 module Poteto
   class PostReview
-    attr_reader :pr_id, :commit_id, :post_lambda, :rate_limit_lambda, :repo
+    attr_reader :commit_id, :repo, :reviewer
+    attr_accessor :post_lambda, :rate_limit_lambda
 
     # def initialize(repo, commit_id, pr_id, post_lambda, rate_limit_lambda: nil)
-    def initialize(config)
-      @commit_id = config.commit_id
-      @pr_id = config.pr_id
-      @repo = config.repo
+    def initialize(reviewer)
+      @commit_id = reviewer.commit_id
+      @pr_id = reviewer.pr_id
+      @repo = reviewer.repo
+      @reviewer = reviewer
     end
 
     def call(reviews)
-      reviews.each do
-        post_lambda.call(config, review)
+      reviews.each do |review|
+        post_lambda.call(reviewer, review)
       end
     end
 
     def post_lambda
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for post_lambda is too high. [<8